test/d2l-applied-filters/d2l-applied-filters.html

```diff
@@ -142,65 +142,79 @@
 					options[8].click();
 
 					afterNextRender(options[1], () => {
-						assert.equal(clearFilters.hidden, false);
+						assert.equal(filterList.hasAttribute('show-clear-list'), true);
 						done();
 					});
 				});
 				test('clear filters button is hidden when 0 filters are applied', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					options[0].deselect();
 					options[3].deselect();
 					options[6].deselect();
 
 					afterNextRender(options[0], () => {
-						assert.equal(clearFilters.hidden, true);
+						assert.equal(filterList.hasAttribute('show-clear-list'), false);
 						done();
 					});
 				});
 				test('clear filters button is hidden when 1 filters is applied', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					options[0].deselect();
 					options[3].deselect();
 
 					afterNextRender(options[0], () => {
-						assert.equal(clearFilters.hidden, true);
+						assert.equal(filterList.hasAttribute('show-clear-list'), false);
 						done();
 					});
 				});
 				test('selecting a 4th filter & clicking clear filters fires a d2l-filter-dropdown-cleared event', function(done) {
-					const clearFilters = appliedFilters.shadowRoot.querySelector('#d2l-clear-filters-button');
+					const filterList = appliedFilters.shadowRoot.querySelector('d2l-labs-multi-select-list');
 					const options = dropdown.querySelectorAll('d2l-filter-dropdown-option');
 
 					const handleClearEvent = () => {
 						document.removeEventListener('d2l-filter-dropdown-cleared', handleClearEvent);
+						assert.equal('true', 'true');
 						done();
 					};
 
 					document.addEventListener('d2l-filter-dropdown-cleared', handleClearEvent);
 
 					options[1].click();
 
-					afterNextRender(clearFilters, () => {
+					afterNextRender(filterList, () => {
+						let clearFilters = undefined;
+						const buttons = filterList.shadowRoot.querySelectorAll('d2l-button-subtle');
+						buttons.forEach(a => {
+							if (a.getAttribute('text') === 'Clear filters' && a.getAttribute('class') !== 'hide') {
```

Comment:
I might prefer an attribute to a class for indicating hidden state (using the `hidden` attribute, maybe) as that feels more resilient to future changes/a public contract, but I do see that's what's being set today. For the class check, I think the best way to do that is `a.classList.contains('hide')`, since more classes may be added in the future